lib/interfaces/rest.js

```diff
@@ -71,15 +73,45 @@ module.exports = function (app) {
             return
           }
         }
-
-        for (var i in path) {
-          var p = path[i]
-
-          if (typeof last[p] !== 'undefined') {
-            last = last[p]
-          } else {
-            next()
-            return
+        if (
+          path.length == 1 &&
+          path[0] === 'vessels' &&
+          req.query.radius &&
+          req.query.latitude &&
+          req.query.longitude
+        ) {
+          debug(`radius query: ${req.query.radius}`)
+
+          var vessels = {}
+
+          _.forIn(data.vessels, (value, key) => {
+            var position = _.get(value, 'navigation.position')
+
+            if (position) {
+              if (
+                geolib.isPointInCircle(
+                  position.value,
+                  req.query,
+                  req.query.radius
+                )
+              ) {
+                debug(`vessel: ${key} ${JSON.stringify(position)}`)
```

Comment:
Very minor here, but as discussed in canboatjs JSON.stringify is surprisingly heavy and I'd like to start the habit of doing this like `debug.enabled && debug(...)`